doc/modules/preprocessing.rst

```diff
@@ -181,9 +166,7 @@ Here is how to use the toy data from the previous example with this scaler::
   array([2.,  1.,  2.])
 
 
-As with :func:`scale`, the module further provides convenience functions
-:func:`minmax_scale` and :func:`maxabs_scale` if you don't want to create
-an object.
+See also :func:`minmax_scale` and :func:`maxabs_scale`.
```

Comment:
Same here, but for `sklearn.preprocessing.MinMaxScaler` and `sklearn.preprocessing.MaxAbsScaler`.

Reply:
Ok, same here